src/MultiExtractor/Extractor.cs

```diff
@@ -278,6 +278,7 @@ private List<FileEntry> ExtractGnuArFile(FileEntry fileEntry)
             catch (Exception e)
             {
                 Logger.Debug("Failed to extract Gnu Ar file {0} {1}", fileEntry.FullPath, e.GetType());
+                throw e;
```

Comment:
Should this just be `throw`?

[StackOverflow post](https://stackoverflow.com/questions/730250/is-there-a-difference-between-throw-and-throw-ex)

Reply:
Good to know!